BrainPortal/app/views/portal/about_us.html.erb

```diff
@@ -37,9 +37,12 @@
   <br>
   <strong>Portal Up Since:</strong> <%= to_localtime(CBRAIN::Startup_LocalTime,:datetime) %> (for: <%= pretty_elapsed((Time.now.localtime - CBRAIN::Startup_LocalTime).to_i) %>)
 
-  <% plugins_basenames = Dir.glob(Pathname.new(CBRAIN::Plugins_Dir) + "cbrain-plugins-*")
-                         .map { |x| Pathname.new(x).basename.to_s }
-                         .reject { |x| x =~ /^cbrain-plugins-base$/ } %>
+  <% plugins_basenames = Pathname(CBRAIN::Plugins_Dir)
```

Comment:
This is perfect, except can we just add `.sort` because the filesystem doesn't return the entries in a deterministic order, and we want to be able to compare the reports from one system to another etc.